src/main/resources/application.properties

```diff
@@ -76,10 +76,13 @@ warn.rbac.slow=true
 warn.rbac.tolerance=1S
 
 quarkus.jaeger.enabled=false
-quarkus.jaeger.service-name=policies-api 
+quarkus.jaeger.service-name=policies-api
 quarkus.jaeger.sampler-type=const
 quarkus.jaeger.sampler-param=1
 
+# OpenTelemetry
+quarkus.opentelemetry.tracer.exporter.otlp.endpoint=
```

Comment:
you should only set properties that have a value... In this case you might be clearing the default endpoint.

Reply:
Is it possible to disable the exporter without disabling the opentracing as whole?

Reply:
not sure what you mean, OpenTracing and OpenTelemetry Tracing are different frameworks and the mentioned property relates to OpenTelemetry.

Reply:
Here the endpoint is checked for presence:
https://github.com/quarkusio/quarkus/blob/a85dbeb7ec2974ef6b18643b3e6a213e13b636e4/extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/OtlpRecorder.java#L24-L25